src/vulkan-renderer/world/cube.cpp

```diff
@@ -213,6 +308,24 @@ void Cube::indent(const std::uint8_t edge_id, const bool positive_direction, con
     m_polygon_cache_valid = false;
 }
 
+void Cube::rotate(const RotationAxis::Type &axis, int rotations) {
+    rotations = ((rotations % 4) + 4) % 4;
```

Comment:
Why two times mod 4?
At least from a mathematical perspective `rotations % 4` should be enough, shouldn't it?

Reply:
I have to catch negative (anti clockwise rotations too) so first mod 4 will reduce it to [-3, 3] and the second (with +4) to [0, 3]

Reply:
Oh, I only fast checked how modulo in python works (thus the question) and as it turns out the implementation is not consistent across languages!